GUI/guiMain.xml

```diff
@@ -105,8 +105,8 @@
 		<element type="button">
 			<attributes>
 				<int name="Id" value="12010" />
-				<string name="Caption" value="Solo" />
-				<rect name="Rect" value="1000, 250, 1260, 320" />
+				<string name="Caption" value="Tutorial" />
```

Comment:
I disagree with this change. This should stay as "Solo" or "Singleplayer" because it leads to a menu with more than just the tutorial.